src/pages/Software.pug

```diff
@@ -2,5 +2,91 @@ extends ../templates/contents.pug
+        To do this, we first need to encode information in form of a base sequence.
+        Even more importantly, we need to be able to decode a sequenced DNA strand to recover the original information.
+        These two tasks are performed by our software.
+        The main focus lies on error correction in the decoding process.
+        Many errors can occur during DNA synthesis and sequencing.
+        By utilizing the peculiarities of our semi-specific synthesis in combination with the capabilities of nanopore sequencing, we will be able to robustly decode information from DNA.
+        This plays an important role in making this technique easier to use and thus accessible to everyone.
+        Our software addresses a problem that is not very common today. However, it has the potential to become a standard tool of bioinformatics when DNA data storage eventually becomes a commonplace technique.
+
+        # Overview
+
+        Our software is written in Python.
+        Right now it consists of a program library and a command line tool which utilizes that library.
+        The command line tool makes all the basic functionalities of our library available, namely encoding and decoding information.
+        In addition, it would be possible to implement a graphical user interface, which would make the usage even easier for everyone.
+
+        In the following, the two functions of our software encoding and decoding are explained in detail.
+        It will become clear how we are able to store information on DNA, although our synthesis approach is semi-specific.
+        We will also discuss the challenges that arise when decoding information.
+
+        # Encoding Information on DNA
+
+    - var imgpath = "" + require("../assets/img/Software/encoding.svg")["default"]
+    +image(1, imgpath, "Overview over the steps needed for encoding data into DNA.")
+
+    :markdown-it(html)
+
+        ## Changes of Nucleotides as Information Carriers
+
+        Our DNA synthesis is semi-specific, meaning that we cannot control how many nucleotides are attached in each step.
+        This means that we will always have chunks of unknown length of each nucleotide, e.g. "AAACCCCCAAAAAGGGG".
+        These chunks are also called homonucleotides.
+        To still be able to encode information in this, we are using a rather unusual encoding scheme.
+        Instead of encoding one bit of information to each base pair, we are encoding information in each change of bases.
+        For example, the change from a homonucleotide of As to a homonucleotide of Cs could encode a `1`.
+
+        ## Using a Ternary Code
+
+        From each homonucleotide, three other homonucleotides are reachable.
+        From "A" we can go to "C", "G", and "T".
+        From "C" we can go to "A", "G", and "T", and so on.
+        This means that to make use of all possible combinations, we do not need a binary code, but a ternary code.
```

Comment:
So, to be able to use all possible combinations, we do not need a binary code, but a ternary code.